src/pcapi/core/users/api.py

```diff
@@ -251,6 +252,28 @@ def update_beneficiary_mandatory_information(
     )
 
 
+def update_user_information_from_external_validator(user: User, data: fraud_models.JouveContent) -> User:
```

Comment:
Je ne suis pas super à l'aise avec le terme validator ici
```suggestion
def update_user_information_from_external_source(user: User, data: fraud_models.JouveContent) -> User:
```